src/client/components/ContactUs.tsx

```diff
@@ -77,24 +77,25 @@ export const ContactUs: React.FC<RouteComponentProps> = () => {
 
     const onSubmit = async (formRef) => {
         // printData(firstName, lastName, phone, email, comment);
-        if (formRef.current.reportValidity()) {
-            const success = await contactUs({variables: {firstName, lastName, phoneNumber, email, comment}});
+        const success = await contactUs({variables: {firstName, lastName, phoneNumber, email, comment}});
```

Comment:
Oh wait we should probably check that the email is valid before making this function call

Reply:
I did, it should be below the highlighted lines